examples/12_optimize_yaw.py

```diff
@@ -74,23 +74,21 @@ def calculate_aep(fi, df_windrose, column_name="farm_power"):
         df_windrose[yaw_cols] = 0.0  # Add zeros
 
     # Derive the wind directions and speeds we need to evaluate in FLORIS
-    wd_array = np.array(df_windrose["wd"].unique(), dtype=float)
-    ws_array = np.array(df_windrose["ws"].unique(), dtype=float)
+    wd_array = np.array(df_windrose["wd"], dtype=float)
+    ws_array = np.array(df_windrose["ws"], dtype=float)
     yaw_angles = np.array(df_windrose[yaw_cols], dtype=float)
     fi.reinitialize(wind_directions=wd_array, wind_speeds=ws_array)
 
     # Map angles from dataframe onto floris wind direction/speed grid
     X, Y = np.meshgrid(wd_array, ws_array, indexing='ij')
```

Comment:
I don't think this line is necessary, since it doesn't appear X and Y are used anywhere.

Reply:
Good catch!